### PR TITLE
[@kadena/transfer] Implementation Debug Library

### PR DESCRIPTION
**Pull request by nil-amrutlal-dept**

### CODE REVIEW

File: packages/apps/transfer/src/pages/transfer/cross-chain-transfer-finisher/index.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ interface IPactResultError {
 }
 
 const CrossChainTransferFinisher: FC = () => {
-  const debug = Debug('XChain-Transfer-Finisher');
+  const debug = Debug('kadena-transfer:pages:cross-chain-transfer-finisher');
```
Comment (MRVDH) [inline packages/apps/transfer/src/pages/transfer/cross-chain-transfer-finisher/index.tsx:52]: `kadena-transfer:pages:transfer:cross-chain-transfer-finisher`

File: packages/apps/transfer/src/pages/transfer/cross-chain-transfer-tracker/index.tsx [hunk 2]
```diff
@@ -41,6 +42,7 @@ const CrossChainTransferTracker: FC = () => {
   const { network } = useAppContext();
   const router = useRouter();
 
+  const debug = Debug('kadena-transfer:pages:cross-chain-transfer-tracker');
```
Comment (MRVDH) [inline packages/apps/transfer/src/pages/transfer/cross-chain-transfer-tracker/index.tsx:45]: `kadena-transfer:pages:transfer:cross-chain-transfer-tracker`

File: packages/apps/transfer/src/services/faucet/index.ts [hunk 3]
```diff
@@ -22,6 +23,8 @@ const FAUCET_PRIVATE_KEY = env(
   '<PROVIDE_FAUCET_PRIVATEKEY_HERE>',
 );
 
+const debug = Debug('kadena-transfer:services:faucet-index');
```
Comment (MRVDH) [inline packages/apps/transfer/src/services/faucet/index.ts:26]: We can leave out the index part imo `kadena-transfer:services:faucet`

**Review by MRVDH — APPROVED** [review]
minor naming changes

**Review by sanderlooijenga — APPROVED** [review]

**Comment by webpro** [comment]
The `debug` lib is kinda frozen. This is not a bad thing at all, it's mature and battle-tested. Still, I think it's good to stay critical and look at alternatives (e.g. to keep using the latest best practices and new Node.js features and performance improvements etc). Would be good to know if you considered anything else?

**Review by rezaerami — APPROVED** [review]

**Comment by nil-amrutlal-dept** [comment]
> 

In this implementation, I did not really consider other alternatives since this library is already being used by some of the applications within the mono repository. I agree that options should be explored and will do so, but for now, I will merge this PR and if a better solution is found, I think its good to change the whole library at once

**Comment by nil-amrutlal-dept** [comment]
> The `debug` lib is kinda frozen. This is not a bad thing at all, it's mature and battle-tested. Still, I think it's good to stay critical and look at alternatives (e.g. to keep using the latest best practices and new Node.js features and performance improvements etc). Would be good to know if you considered anything else?

In this implementation, I did not really consider other alternatives since this library is already being used by some of the applications within the mono repository. I agree that options should be explored and will do so, but for now, I will merge this PR and if a better solution is found, I think its good to change the whole library at once

**Pull request merged**